src/pages/Dashboard.jsx

```diff
@@ -0,0 +1,312 @@
+import {
+  Box,
+  Flex,
+  Text,
+  DropdownMenu,
+  DropdownMenuTrigger,
+  DropdownMenuContent,
+  DropdownMenuSeparator,
+  DropdownMenuItem,
+  Button,
+} from "@sparrowengg/twigs-react";
+import {
+  Popover,
+  PopoverTrigger,
+  PopoverArrow,
+  PopoverContent,
+} from "@sparrowengg/twigs-react";
+
+import { CalendarIcon } from "../Components/Icons";
+
+import LeftArrow from "./../asset/leftArrow.svg";
+import moonIconDashboard from "./../asset/moonSvgDemo.svg";
+import Plus from "./../asset/plus.svg";
+import plusLight from "./../asset/plusLight2.svg";
+import AlertCircle from "./../asset/alert-circle.svg";
+import AlertCircle2 from "./../asset/alert-circle2.svg";
+import HorizontalDots from "./../asset/more-horizontal.svg";
+import HorizontalDotsDark from "./../asset/more-horizontalDark.svg";
+
+import { useState } from "react";
+import AreaChartComp from "../Components/AreaChart";
+import PieBarContainer from "../Components/PieBarContainer";
+import { LIGHT_MODE } from "../redux/light-dark/lightDarkTypes";
+import { useDispatch, useSelector } from "react-redux";
+import { parseDate } from "@internationalized/date";
+import {
+  SET_DATE_RANGE,
+  SET_COUNT,
+} from "../redux/fake-api-data/fakeApiDataTypes";
+
+import { CalendarRange } from "@sparrowengg/twigs-react";
+import { COUNT_ABOVE, COUNT_BELOW } from "../utils/data";
+
+const Dashboard = () => {
+  const [value, setValue] = useState({
+    start: parseDate(new Date().toISOString().substring(0, 10)),
+    end: parseDate(new Date().toISOString().substring(0, 10)),
+  });
+
+  const dispatch = useDispatch();
+  const mode = useSelector((store) => store.lightDARK_MODE.lightDARK_MODE);
```

Comment:
Use better `key` names in the store and in the combineReducer, such that we can have a meaningful dot operations, while retrieving the values from the store.